operator/controllers/manifest_controller.go

```diff
@@ -71,13 +71,12 @@ type ManifestReconciler struct {
 	Codec                   *v1alpha1.Codec
 }
 
-const configReadError = "reading install config resulted in an error"
-
 //+kubebuilder:rbac:groups=component.kyma-project.io,resources=manifests,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=component.kyma-project.io,resources=manifests/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=component.kyma-project.io,resources=manifests/finalizers,verbs=update
 //+kubebuilder:rbac:groups="",resources=events,verbs=create;patch;get;list;watch
 //+kubebuilder:rbac:groups="",resources=secrets,verbs=get;list;watch
+//+kubebuilder:rbac:groups="",resources=configmaps,verbs=get;list;watch;update;patch;delete
```

Comment:
we just need get, list, watch

Reply:
But the operator should be able to manage those configmaps, right? like verify before deletion?